README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
```

Comment:
I'm not sure what's happening here, but maybe you could make this line clearer for ¥our users.

Reply:
What do you mean by that?

Reply:
I'll check it out

Reply:
This looks good.